### PR TITLE
few little improvements to the booster release scripts

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

**Comment by Ladicek** [comment]
Depends on https://github.com/thorntail-examples/booster-license-generator/pull/6

File: README.md [hunk 1]
```diff
@@ -1,7 +1,185 @@
-# Booster Release Scripts
+# Release Instructions
```
Comment (sberyozkin) [inline README.md:1]: Good idea to put into the README

File: README.md [hunk 2]
```diff
@@ -1,7 +1,185 @@
...
+├── health-check
+├── health-check-redhat
+├── messaging-work-queue
+├── rest-http
+├── rest-http-crud
+├── rest-http-crud-redhat
+├── rest-http-redhat
+├── rest-http-secured
+└── rest-http-secured-redhat
+```
```
Comment (sberyozkin) [inline README.md:28]: May be add the istio-tracing in this tree ?

Comment (Ladicek) [reply README.md:28]: I didn't add it to the `clone-here.sh` script yet :-)

File: README.md (hunk 1 above)
Comment (Ladicek) [reply README.md:1]: Unfortunately Git didn't detect the rename because I also did a lot of changes.

File: README.md (hunk 2 above)
Comment (sberyozkin) [reply README.md:28]: Sure, I guess it can be done when the `-redhat` version of this example is added

**Review by sberyozkin — APPROVED** [review]

**Pull request merged**